contracts/BlastERC20RebasingYield.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {BlastNativeYield} from "./BlastNativeYield.sol";
+import {IERC20Rebasing, YieldMode} from "./interfaces/IERC20Rebasing.sol";
+
+/**
+ * @title BlastERC20RebasingYield
+ * @notice This contract is a base contract for inheriting functions to claim Blast WETH or USDB yield
+ * @author LooksRare protocol team (👀,💎)
+ */
+contract BlastERC20RebasingYield is BlastNativeYield {
+    address public immutable WETH;
+    address public immutable USDB;
+
+    /**
+     * @param _blast Blast precompile
+     * @param _blastPoints Blast points
+     * @param _blastPointsOperator Blast points operator
+     * @param _governor The address that’s allowed to claim the contract’s yield and gas
+     * @param _usdb USDB address
+     * @param _weth WETH address
+     */
+    constructor(
+        address _blast,
+        address _blastPoints,
+        address _blastPointsOperator,
+        address _governor,
+        address _usdb,
+        address _weth
+    ) BlastNativeYield(_blast, _blastPoints, _blastPointsOperator, _governor) {
+        WETH = _weth;
+        USDB = _usdb;
+
+        IERC20Rebasing(_weth).configure(YieldMode.CLAIMABLE);
+        IERC20Rebasing(_usdb).configure(YieldMode.CLAIMABLE);
+    }
+
+    /**
+     * @notice Claim Blast yield. Guarding of the function is dependent on the inherited contract.
```

Comment:
Also good to state that just inheriting the contract does not allow claiming. Will need to implement a public or external function that calls this internal.

This is often an issue that people who use other libraries that only give an internal function like pausable, only to find out that they cannot access the inherited internal function

Reply:
I will add "Inheriting does not allow claiming by default."
Should I also add "A public or external function is required to access the Claim function" or is that overkill?

Reply:
https://github.com/LooksRare/contracts-libs/pull/103/commits/602487ee6f545070007f6906a929068c28c42f88
I changed this from "Claim" to "_claim" here: https://github.com/LooksRare/contracts-libs/pull/103/commits/534d0539f114da5905628eb7d036ac3cbdb3c38d

Reply:
sorry but can you split the line into multiple lines it's too long

Reply:
https://github.com/LooksRare/contracts-libs/pull/103/commits/763c63da6e72657a1d8e102db78f6021a01f10df